commit d9e8f47c8339682f26650f63975c15f0cd1fb846 (refs/changes/48/2948/1)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-11-15T13:41:05-08:00 (1 year, 11 months ago)
    
    joyent/manta-thoth#152 thoth's user ID can change, breaking sdc-thoth

diff --git a/bin/sdc-thoth-install b/bin/sdc-thoth-install
index 3bddbcd..3c0207c 100755
--- a/bin/sdc-thoth-install
+++ b/bin/sdc-thoth-install
@@ -21,9 +21,9 @@ function onexit
 {
 	if [[ -f $backup ]]; then
 		cp $backup $hosts
-		rm $backup	
+		rm $backup
 	fi
-	
+
 	[[ $1 -ne 0 ]] || exit 0
 	fatal "error exit status $1"
 }
@@ -72,7 +72,7 @@ if [[ -z $MANTA_URL ]]; then
 		MANTA_URL=https://$server
 		insecure=
 	fi
-	
+
 	export MANTA_URL
 fi
 
@@ -92,7 +92,7 @@ if [[ -z $SDC_URL ]]; then
 		if ( ! ping $server 1>&2 ); then
 			fatal "could not ping $server"
 		fi
-	fi	
+	fi
 
 	SDC_URL=https://$server
 	export SDC_URL
@@ -194,8 +194,9 @@ cat > $postboot <<EOF
 
 set -o xtrace
 
-useradd -c "Thoth user" -s /bin/bash -d $home $user
+useradd -u 100 -c "Thoth user" -s /bin/bash -d $home $user
 passwd -u $user
+chown -R $user $home
 cat $crontab | su thoth -c crontab
 
 #
@@ -212,7 +213,7 @@ chmod +x $postboot
 echo Creating SMF manifest ...
 
 #
-# Now create the manifest 
+# Now create the manifest
 #
 mkdir -p `dirname $manifest` || true
 cat > $manifest <<EOF
